README.md

```diff
@@ -0,0 +1,57 @@
+# Super-List
+
+The learning goals of this project are to practice using ES6 standards in Javascript and to implement Webpack.  This project is recreating part of a, now defunct, todolist app called minimalist.
+
+## Buit With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+## Live Demo
+
+[Super List](https://crclothier.github.io/Super-List/)
+
```

Comment:
great job on the project 👍 in order to make your project look even more professional please make sure to link your project live page, in this project you can link the ([dist](https://crclothier.github.io/Super-List/dist)) folder, now on the live demo its redirected to readme file.